cql3/statements/cf_prop_defs.cc

```diff
@@ -322,7 +322,11 @@ void cf_prop_defs::apply_to_builder(schema_builder& builder, schema::extensions_
 
     // for extensions that are not altered, keep the old ones
     auto& old_exts = builder.get_extensions();
-    schema_extensions.insert(old_exts.begin(), old_exts.end());
+    for (auto& [key, ext] : old_exts) {
+        if (!_properties.count(key)) {
```

Comment:
nitpick: C++20 finally introduced `std::unordered_map::contains()`, which does the same thing but sounds less weird

Reply:
Yes, but this is not available in 2019.1, to where this needs to be backported.